lib/AST/Decl.cpp

```diff
@@ -1132,6 +1132,94 @@ VarDecl *PatternBindingDecl::getSingleVar() const {
   return nullptr;
 }
 
+/// Check whether the given type representation will be
+/// default-initializable.
+static bool isDefaultInitializable(TypeRepr *typeRepr) {
```

Comment:
This and the `Pattern *` below could be using `const`.

Reply:
Will fix before merging. Thanks for taking the time to answer all of my questions — I appreciate the patience. 😄